input/intro-notes/StructureDefinition-au-patient-intro.md

```diff
@@ -15,5 +15,38 @@
   - If `Patient.communication.preferred` has not been included, or is set to "false", systems should understand this as the language for the interpreter service is not known.
 - See each Identifier profile page for guidance related to that identifier type.
 
+**Person Recorded Sex Or Gender implementation guidance**
+  This profile supports patient sex and gender information currently in use in various official documents and services using the [Person Recorded Sex Or Gender](https://hl7.org/fhir/extensions/5.1.0-snapshot1/StructureDefinition-individual-recordedSexOrGender.html) extension, as per the guidance in the following table.
+  <table border="1">
+    <thead>
+    <tr>
+    <th>Sex and gender information to represent</th>
+    <th>extension.value</th>
+    <th>extension:type</th>
+    <th>extension:sourceDocument</th>
+    <th>extension:sourceField</th>
+    <th>extension:jurisdiction</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+    <td>Sex and gender information in use in documents, e.g. Australian passport or driver's license</td>
+    <td> </td>
```

Comment:
I read this guidance is stating to leave the extension value blank. Since that field is the actual sex or gender value I do not agree. These cells cannot remain blank.

Reply:
Updated.

Reply:
This has been added but minor tweak instead of "A sex or gender ..." say "The sex or gender ..."

Reply:
Done